content/en/docs/concepts/configuration/overview.md

```diff
@@ -69,6 +69,7 @@ A Service can be made to span multiple Deployments by omitting release-specific
 
 A desired state of an object is described by a Deployment, and if changes to that spec are _applied_, the deployment controller changes the actual state to the desired state at a controlled rate.
 
+- Use the [Kubernetes common labels](/docs/concepts/overview/working-with-objects/common-labels/) for common use cases. These standardized labels enrich the metadata in a way that allows tools, including `kubectl` and [dashboard](/docs/tasks/access-application-cluster/web-ui-dashboard), to work in an interoperable way. It also enhances the capability to query objects using a common vernacular.
```

Comment:
Hi @rjain21. Thanks for opening this pull request.
I think this paragraph could list a couple of specific reasons to use common labels.
Could you replace or remove this phrase "using a common vernacular"?

Use [Kubernetes common labels and annotations](/docs/concepts/overview/working-with-objects/common-labels/) to categorize objects. Common labels allow tools, including `kubectl` and [dashboard](/docs/tasks/access-application-cluster/web-ui-dashboard), to manage objects in an interoperable way. You can query objects ...

Reply:
Documentation changed as suggested. I agree that the vernacular word is too fancy, but putting more use cases here in a "best practice" document is too much detail. Therefore, I have changed the language a little as you suggested.